.github/workflows/review-check.yml

```diff
@@ -0,0 +1,56 @@
+name: Enforce Review Rules
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  enforce-review-rules:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Get list of files changed
+      id: files
+      run: |
+        echo "::set-output name=files::$(git diff --name-only ${{ github.event.before }} ${{ github.sha }} | tr '\n' ' ')"
+    
+    - name: Check if PR modifies specific path
+      id: check-path
+      run: |
+        if echo "${{ steps.files.outputs.files }}" | grep -q '^src/'; then
+          echo "::set-output name=specific_path::true"
+        else
+          echo "::set-output name=specific_path::false"
+        fi
+
+    - name: Ensure required reviewers
+      id: ensure-reviewers
+      run: |
+        GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
+        PR_NUMBER=${{ github.event.pull_request.number }}
+        REPO=${{ github.repository }}
+        
+        # Fetch reviewers of the PR
+        REVIEWERS=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
+                    "https://api.github.com/repos/$REPO/pulls/$PR_NUMBER/reviews" \
+                    | jq -r '.[].user.login')
```

Comment:
For extra safety I recommend setting `shell: bash` after `name:` so that `set -o pipefail` is [enabled by default](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference), and then if curl fails the run fails as well. (Admittedly, `jq` is going to fail as well in most cases with an error curl output, but you never know.)